### DISCUSSION TITLE
Ascii images not displaying correctly.

### DISCUSSION THREAD

**Discussion by GameLord2011** [lptstr/winfetch#228] [Q&A]
I am trying to make my terminal look op, and the ascii art version of my image renders incorrectly.
[The image](https://gamelord2011.github.io/assets/icons/pwsh-preview.scale-200.png)
![Screenshot 2024-12-13 190455](https://github.com/user-attachments/assets/b086c619-abf0-42f1-8b0f-e44e20f5323c)

**Comment by rashil2000** [comment]
How else do you want it to look?

**Reply by GameLord2011** [reply]
It is not converting the png file into ascii art correctly

**Reply by rashil2000** [reply]
Could you tell what the correct ascii art looks like?

**Reply by GameLord2011** [reply]
Like the top ansi block art, but with ascii characters

**Reply by rashil2000** [reply]
There's no way to customize the look of ASCII art, sorry! You can put a custom ASCII image in the [config file](https://github.com/lptstr/winfetch/wiki/Configuration) if you want.

**Reply by GameLord2011** [reply]
Explaination: config has option for cusom image for ascii art, and boolean to change from ansi-block to ascii-chars.
